index.html

```diff
@@ -47,10 +47,154 @@ <h1>Hey there. I'm Sodiq. <br> <span>I'm a Software Developer</span></h1>
+
+      <div class="recent-works-cards">  <!-- flexbox item-->
+        <div class="works work1"> <!-- flexbox container-->
+          <div class="work-details"> <!--Grid container-->
+            <p>Multi-Post Stories Gain+Glory</p> 
+            <ul class="language-tags"> <!--flex container-->
+              <li>Ruby on Rails</li>
+              <li>CSS</li>
+              <li>Javascript</li>
+              <li>HTML</li>
+            </ul>
+            <a href="#" class="works-button">See Project</a>
+          </div>
+        </div>
+
+        <div class="works work2">
+          <div class="work-details"> <!--Grid container-->
+            <p>Multi-Post Stories Gain+Glory</p> 
+            <ul class="language-tags"> <!--flex container-->
+              <li>Ruby on Rails</li>
+              <li>CSS</li>
+              <li>Javascript</li>
+              <li>HTML</li>
+            </ul>
+            <a href="#" class="works-button">See Project</a>
+          </div>
+        </div>
+
+        <div class="works work3">
+          <div class="work-details"> <!--Grid container-->
+            <p>Multi-Post Stories Gain+Glory</p> 
+            <ul class="language-tags"> <!--flex container-->
+              <li>Ruby on Rails</li>
+              <li>CSS</li>
+              <li>Javascript</li>
+              <li>HTML</li>
+            </ul>
+            <a href="#" class="works-button">See Project</a>
+          </div>
+        </div>
+
+        <div class="works work4">
+          <div class="work-details"> <!--Grid container-->
+            <p>Multi-Post Stories Gain+Glory</p> 
+            <ul class="language-tags"> <!--flex container-->
+              <li>Ruby on Rails</li>
+              <li>CSS</li>
+              <li>Javascript</li>
+              <li>HTML</li>
+            </ul>
+            <a href="#" class="works-button">See Project</a>
+          </div>
+        </div>
+
+        <div class="works work5">
+          <div class="work-details"> <!--Grid container-->
+            <p>Multi-Post Stories Gain+Glory</p> 
+            <ul class="language-tags"> <!--flex container-->
+              <li>Ruby on Rails</li>
+              <li>CSS</li>
+              <li>Javascript</li>
+              <li>HTML</li>
+            </ul>
+            <a href="#" class="works-button">See Project</a>
+          </div>
+        </div>
+
+        <div class="works work6">
+          <div class="work-details"> <!--Grid container-->
+            <p>Multi-Post Stories Gain+Glory</p> 
+            <ul class="language-tags"> <!--grid container-->
+              <li>Ruby on Rails</li>
+              <li>CSS</li>
+              <li>Javascript</li>
+              <li>HTML</li>
+            </ul>
+            <a href="#" class="works-button">See Project</a>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section class="about-me">
+
+      <div class="about-me-text">
+        <p class="about-me-title">About me</p>
```

Comment:
- [ ] The title 'About me' should uses a header tag `hx` (h1, h2,...). Kindly, fix it! 
"_you're almost there, the little details make the beauty of things_"